test/declaration_test.ts

```diff
@@ -381,7 +381,7 @@ abstract class X {
 }`);
   });
   it('supports interface properties', () => {
-    expectTranslate('interface X { x: string; y; }').to.equal(`@anonymous
+    expectTranslate(`interface X { 'x': string; y; }`).to.equal(`@anonymous
```

Comment:
I would recommend adding a second
expectTranslate clause instead of modifying this one.
"there are no bad tests"
In general any time existing tests weren't covering enough cases err on the side of adding extra tests instead of modifying existing ones.